community/values/src/main/java/org/neo4j/values/StructureBuilder.java

```diff
@@ -31,6 +32,10 @@
 
     static <T> T build( StructureBuilder<AnyValue,T> builder, MapValue map )
     {
+        if ( map.size() == 0 )
+        {
+            throw new InvalidTemporalArgumentException( "At least one temporal unit must be specified." );
```

Comment:
What message did we get before (just out of curiosity)?

Reply:
`Builder state empty`, I don't think that was a very helpful error message.